src/components/productDetails/card.jsx

```diff
@@ -18,77 +18,69 @@ const Card = ({
   let [showModal, setShowModal] = useState(false);
 
   return (
-    <div className="column is-2">
-      <div className="card">
-        <div>
-          {showModal === true ? (
-            <div>
-              <Modal
-                onExitClick={() => {
-                  setShowModal(false);
-                }}
-                title="Comparing"
-              >
-                <ComparisonTable product={product} pageProduct={pageProduct} />
-              </Modal>
-            </div>
+    <div className="card">
+      <div>
+        {showModal === true ? (
+          <div>
+            <Modal
```

Comment:
this Modal component is a getting a ton of use! Good call on making a basic container for it that we can all use!

Reply:
Thanks for the feedback I figured it was a good concept to learn as well.